perma-payments/perma_payments/views.py

```diff
@@ -279,7 +279,7 @@ def update(request):
 @sensitive_post_parameters(*SENSITIVE_POST_PARAMETERS)
 def cybersource_callback(request):
     """
-    In dev, curl http://192.168.99.100/cybersource-callback/ -X POST -d '@/Users/rcremona/code/perma-payments/sample_response.txt'
+    In dev, curl http://192.168.99.100/cybersource-callback/ -X POST -d 'path/to/sample_response.txt'
     """
```

Comment:
sample_response.txt is missing

Reply:
Aha, comment out of date! There are samples here:
https://github.com/harvard-lil/perma-payments/blob/develop/sample_response_subscribe.txt and 
https://github.com/harvard-lil/perma-payments/blob/develop/sample_response_update.txt,
but we can't curl like the above, because the timestamps will be invalid....

Reply:
I'll update the comment to that effect.